### DISCUSSION TITLE
[Feedback] Map display at the top of the Enterprise audit log page is being deprecated

### DISCUSSION THREAD

**Discussion by boylejj** [community/community#43979] [Enterprise]
## What is happening?

The map display at the top of the Enterprise audit log will be deprecated. The map is driving performance challenges for the overall Enterprise audit log page, causing an increased frequency of page timeouts.

<img width="1003" alt="image" src="https://user-images.githubusercontent.com/95828167/211847185-388c1a2f-5b85-4fb1-9de7-132d53922b0f.png">

## Why is this happening
The map is driving performance challenges for the overall Enterprise audit log page, causing an increased frequency of page timeouts. We have observed that the map is used in 0.01% of all queries to the enterprise audit log, with such low usage we feel the adverse performance consequences seem to outweigh the display's utility. 

## What to do with feedback?
Sometimes, numbers do not tell the whole story. So, if this is causing you pain, we want to hear from you. If you have feedback on this change, please share that feedback in this community discussion. The audit log team will monitor this discussion to understand and help mitigate any adverse consequences you're experiencing.

**Comment by agonzalez-plume** [comment]
-1 for dropping the chart, I thought it was interesting feature. We used it every so often. Understand the issue and appreciate the description why it's going away.
